lib/accountregistry.cpp

```diff
@@ -15,6 +21,7 @@ void AccountRegistry::add(Connection* a)
     beginInsertRows(QModelIndex(), size(), size());
     push_back(a);
     endInsertRows();
+    Q_EMIT accountCountChanged();
```

Comment:
```suggestion
    emit accountCountChanged();
```
We only use `Q_SIGNALS` and `Q_SLOTS` in header files because they are/can be included into other projects.